test/rest_clients/usersclient.spec.js

```diff
@@ -138,21 +138,27 @@ describe('Users Rest Client', function() {
             .catch(done);
     });
 
-    // TODO: authenticate guest (current) to be able to add new users
     it('should add a new user', function(done) {
+
+        let justAddedBody = {
+            password: 'pass',
+            email: 'justAdded@example.com',
+            canCreate: true
+        }
+
         rest.users.getAllUsers()
             .then(function(usersList) {
                 logger.debug('Initial: ', usersList);
             })
             .then(function() {
-                return rest.users.addUser('justAdded', {_id: 'justAdded', email: 'just@added.com'});
+                return rest.users.addUser('justAdded', justAddedBody);
             })
             .then(function() {
-                return rest.users.getUserData('justAdded');
+                return rest.users.getUser('justAdded');
             })
-            .then(function(userData) {
-                logger.debug('User data:', userData);
-                expect(userData).email.to.deep.equal('just@added.com');
+            .then(function(user) {
+                logger.debug('User data:', user);
+                expect(user.email).to.deep.equal('justAdded@example.com');
```

Comment:
I would also suggest to use .to.deep.equal only if you check objects, but in case of primitive data-types .to.equal should be fine